### ISSUE TITLE
"Oops, this parser is just broken"

### ISSUE THREAD

**Issue by jenskutilek** [googlefonts/fontc#801]
I tried to compile my [Sudo](https://github.com/jenskutilek/sudo-font/tree/master/src) font with `fontc`, but it fails without any specifics:

```
fontc Sudo.glyphs
Font IR error: 'Unable to parse "Sudo.glyphs": Unable to read glyphs file: Unable to parse Sudo.glyphs: Oops, this parser is just broken'
```

I [converted](https://github.com/jenskutilek/glyphspkg) the original `.glyphspackage` file to `.glyphs`, attached.

[Sudo.glyphs.zip](https://github.com/googlefonts/fontc/files/15332330/Sudo.glyphs.zip)

**Comment by anthrotype** [comment]
thanks. Could you try with `RUST_BACKTRACE=1` env variable and past here what you get?

**Comment by anthrotype** [comment]
actually RUST_BACKTRACE=1 would not work in this case as this is not a panic. RUST_LOG=debug maybe

**Comment by anthrotype** [comment]
I put a few println! before the SomethingWentWrong error which gives the Ooops, and I think it's the "disabled = 1" key in some of the custom parameters that is making it fail. So the glyphs-reader parser currently doesn't support disabled custom parameters. Looks like it's a known issue https://github.com/googlefonts/fontc/issues/312

If I remove those disabled custom params, I get another error but probably better to file a separate issue for that.

**Comment by anthrotype** [comment]
ideally we should support parsing disabled custom params, but at this generic error message should be made clearer, more specific

**Comment by jenskutilek** [comment]
Thank you for testing!

**Comment by rsheeter** [comment]
Thank you for the report

**Comment by cmyr** [comment]
this is fixed in #880
